src/chart.ts

```diff
@@ -60,12 +79,32 @@ export class Chart extends Construct {
    */
   private readonly _labels?: { [name: string]: string };
 
+  /**
+   * Whether or not this chart includes construct metadata.
+   */
+  public readonly includesConstructMetadata: boolean;
+
   constructor(scope: Construct, id: string, props: ChartProps = { }) {
     super(scope, id);
     this.namespace = props.namespace;
     this._labels = props.labels ?? {};
 
     Object.defineProperty(this, CHART_SYMBOL, { value: true });
+
+    this.includesConstructMetadata = props.constructMetadata ?? (process.env.CDK8S_CONSTRUCT_METADATA === 'true' ? true : false);
```

Comment:
The idea behind allowing env variable config is so that this can be explicitly turned on by the CLI when synthesis is executed with a validation.